commcare_connect/opportunity/migrations/0066_exchangerate_exchangerate_unique_currency_code_date.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.5 on 2025-01-09 13:24
+import logging
+
+from django.db import migrations, models
+from django.db.models.functions import TruncDate
+
+from commcare_connect.opportunity.visit_import import get_exchange_rate
+
+logger = logging.getLogger(__name__)
+
+
+def update_exchange_rate(apps, schema_editor):
+    Payment = apps.get_model("opportunity.Payment")
+
+    payments = (
+        Payment.objects.annotate(date_only=TruncDate("date_paid"))
+        .values(
+            "id", "date_only", "amount", "opportunity_access__opportunity__currency", "invoice__opportunity__currency"
+        )
+        .distinct()
+    )
+
+    for payment in payments:
+        date_paid = payment["date_only"]
+        currency = payment["opportunity_access__opportunity__currency"] or payment["invoice__opportunity__currency"]
+
+        if currency is "USD":
+            exchange_rate = 1
+        else:
+            exchange_rate = get_exchange_rate(currency, date_paid)
+            logger.info(
```

Comment:
This logger is misleading right? This never actually changes the values? Or sets them when they are missing?